tests/test_data/invalid/product.json

```diff
@@ -0,0 +1,8 @@
+{
+    "agents": [],
+    "target": {
+        "key": "appVersion",
+        "version": "0.98.4"
+    },
+    "forceLowerCase": false
```

Comment:
Should we have a test case for case insensitive as well? Can be a quick test, with a `product.json` defined inside the test case.